main.cpp with file reading, 2 input test files

saif9817 opened the pull request:
The first requested change to add file reading for the input, added an extra portion to default to a certain filename if no name is provided, have provided 2 test input files with a single change in the 2nd input, there are print statements commented out that were used for testing the functionality of the default and provided filename feature

saif9817 committed:
main.cpp with file reading, 2 input test files

The first requested change to add file reading for the input, added an extra portion to default to a certain filename if no name is provided, have provided 2 test input files with a single change in the 2nd input, there are print statements commented out that were used for testing the functionality of the default and provided filename feature

cruizm commented on top_model/top_model/main.cpp:
```diff
@@ -0,0 +1,175 @@
+		inputReader.open(filePath);
+	}
+	
+	if(inputReader.is_open()){
```
Hello Saif,
This is good. Can you please modify it in such a way that in the name of the file we have the name of the input parameter, so everything is less error-prone?
Example:
mortality = 10.5
infectivity period = 15
...

saif9817 committed:
main.cpp, input.txt, data.txt

The new main.cpp now allows users to have tags for the data visible in the input text file, the 2 separate files, input (the default) and data (can be used by added filename when launching program) are for testing purposes

saif9817 commented:
I have added the functionality to parse through tags, choosing to leave it in the exact order, I have added the assert to mortality only for testing, if you run the program with data.txt (which has an empty mortality field) it aborts the program but the message is not showing for me

saif9817 committed:
main.cpp, 2 input files

Fixed assert to provide specific error information

saif9817 commented:
I have fixed the earlier issue, it is now providing specific errors

saif9817 committed:
main.cpp, README for input.txt, template.txt

Added comments to main.cpp, added descriptive readme for user input, added blank template file for the input

saif9817 commented:
Hi Cristina, please find the newly added comments for the code as well as a readme for the user input section and a template for users, please let me know if you need me to make any other changes

cruizm committed:
Merge branch 'master' into read-from-text
